common/client-libs/validator-client/src/nymd/mod.rs

```diff
@@ -939,6 +950,18 @@ impl<C> NymdClient<C> {
         )
     }
 
+    #[execute("vesting")]
+    fn _vesting_update_locked_pledge_cap(
```

Comment:
Do we even need to have the ability to dynamically update it? From what I understood we set it once and leave it like that

Reply:
I've got a feeling you might be right : )